default/python/AI/FreeOrionAI.py

```diff
@@ -138,9 +138,12 @@ def prepareForSave():  # pylint: disable=invalid-name
     print "Preparing for game save by serializing state"
 
     # serialize (convert to string) global state dictionary and send to AI client to be stored in save file
-    dump_string = pickle.dumps(foAIstate)
-    print "foAIstate pickled to string, about to send to server"
-    fo.setSaveStateString(dump_string)
+    try:
+        dump_string = pickle.dumps(foAIstate)
+        print "foAIstate pickled to string, about to send to server"
+        fo.setSaveStateString(dump_string)
+    except:
+        print "foAIstate unable to picke string"
```

Comment:
OK, 16b93b9 fixes the cited problem.   Catching any exceptions here is probably still a good thing, so long as there is at least a louder error message like I propose.